pddlgym/core.py

```diff
@@ -417,11 +495,30 @@ def sample_transition(self, action):
 
         return state, reward, done, debug_info
 
+    def sample_transition(self, action):
+        state = self._get_successor_state(self._state, action, self.domain,
+                                          inference_mode=self._inference_mode,
+                                          raise_error_on_invalid_action=self._raise_error_on_invalid_action)
+        return self._get_new_state_info(state)
+
     def _get_successor_state(self, *args, **kwargs):
         """Separated out to allow for overrides in subclasses
         """
         return get_successor_state(*args, **kwargs)
 
+    def _get_successor_states(self, *args, **kwargs):
```

Comment:
Don't really know if this is necessary, but I followed the format in `_get_successor_state`.

Reply:
Looks great!